src/js/store/configureStore.js

```diff
@@ -4,12 +4,17 @@ import thunk from 'redux-thunk';
 import { createLogger } from 'redux-logger'
 
 export default function configureStore(initialState) {
-  const logger = createLogger()
+  let logger = createLogger()
+  let middleware = applyMiddleware(thunk)
+
+  if (process.ENV !== 'production') {
+    middleware = applyMiddleware(thunk, logger)
+  }
```

Comment:
This function looks like its responsibility can be further reduced. We can have a separate function that returns the right middleware depending on the environment.